packages/node_modules/@webex/plugin-authorization-browser/src/authorization.js

```diff
@@ -181,7 +181,8 @@ const Authorization = WebexPlugin.extend({
    * @returns {Promise}
    */
   requestAccessTokenFromJwt({jwt}) {
-    const hydraUri = this.webex.internal.services.get('hydra', true) ||
+    const hydraUri = this.webex.internal.services.get('hydra', true) ?
```

Comment:
Can the code be more detailed for ease of understanding? The `||` combined with `?` makes this difficult to understand.

Reply:
I have appended a comment to help expand on what is occurring.